src/pages/ReimbursementAccount/NonUSD/BeneficialOwnerInfo/BeneficialOwnersList.tsx

```diff
@@ -0,0 +1,115 @@
+    const styles = useThemeStyles();
+    const {isOffline} = useNetwork();
+
+    const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT);
+    const [reimbursementAccountDraft] = useOnyx(ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM_DRAFT);
+    const ownershipChartValue = reimbursementAccount?.achData?.additionalData?.corpay?.[ENTITY_CHART] ?? reimbursementAccountDraft?.[ENTITY_CHART] ?? [];
+
+    const policyID = reimbursementAccount?.achData?.policyID ?? '-1';
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`);
+    const currency = policy?.outputCurrency ?? '';
+
+    const owners =
+        reimbursementAccountDraft &&
+        ownerKeys.map((ownerKey) => {
+            const ownerData = getValuesForBeneficialOwner(ownerKey, reimbursementAccountDraft);
+
+            return (
+                <MenuItem
+                    key={ownerKey}
+                    title={`${ownerData.firstName} ${ownerData.lastName}`}
+                    description={`${ownerData.street}, ${ownerData.city}, ${ownerData.state} ${ownerData.zipCode}`}
+                    wrapperStyle={[styles.ph5]}
+                    icon={Expensicons.FallbackAvatar}
+                    iconType={CONST.ICON_TYPE_AVATAR}
+                    onPress={() => {
+                        handleOwnerEdit(ownerKey);
+                    }}
+                    iconWidth={40}
+                    iconHeight={40}
+                    interactive
+                    shouldShowRightIcon
+                    displayInDefaultIconColor
+                />
+            );
+        });
+
+    return (
+        <SafeAreaConsumer>
+            {({safeAreaPaddingBottomStyle}) => (
+                <ScrollView
+                    style={styles.pt0}
+                    contentContainerStyle={[styles.flexGrow1, styles.ph0, safeAreaPaddingBottomStyle]}
+                >
+                    <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5]}>{translate('beneficialOwnerInfoStep.letsDoubleCheck')}</Text>
+                    <Text style={[styles.p5, styles.textSupporting]}>{translate('beneficialOwnerInfoStep.regulationRequiresUsToVerifyTheIdentity')}</Text>
+                    {owners !== undefined && owners?.length > 0 && (
+                        <View>
+                            <Text style={[styles.textSupporting, styles.pv1, styles.ph5]}>{`${translate('beneficialOwnerInfoStep.owners')}:`}</Text>
+                            {owners}
+                        </View>
+                    )}
+                    {currency === CONST.CURRENCY.AUD && (
+                        <MenuItemWithTopDescription
+                            description={translate('ownershipInfoStep.certified')}
+                            title={ownershipChartValue.map((file) => file.name).join(', ') || ''}
+                            shouldShowRightIcon
+                            onPress={handleOwnershipChartEdit}
+                            style={[styles.mt8]}
```

Comment:
I think we won't need this mt8 right?
```suggestion
```

If so we have this big gap between MenuItem and the text above:

<img width="374" alt="Screenshot 2024-11-18 at 15 19 28" src="https://github.com/user-attachments/assets/e2bffbdd-75f2-4d12-b629-75390e11fa14">

Reply:
<img width="358" alt="image" src="https://github.com/user-attachments/assets/0504468c-dd9d-42fb-8b1b-a7c7ffadb7e2">
We have such big gap when owner's list is present. But I guess if there are no owners it can be smaller so it doesn't look weird. Ill fix it